src/v2/guide/filters.md

```diff
@@ -26,30 +27,61 @@ filters: {
 }
 ```
 
-or define a filter globally:
+hoặc định nghĩa ở cấp toàn cục:
```

Comment:
Theo em nghĩ là "định nghĩa Filter toàn cục"

Reply:
Nếu em đọc cả câu thì sẽ hiểu hơn. Ngoài ra câu của em sẽ dịch thành “define a global filter.”